### DISCUSSION TITLE
Stepmania 5.0 on Arch Linux doesn't go above 60 FPS

### DISCUSSION THREAD

**Discussion by puyoxyz** [stepmania/stepmania#2105] [Q&A] [answered]
Hello, I am trying to play StepMania 5.0 (installed from the AUR, version 5.0-git-b91dc72) but my FPS (checked using the render stats from the F3 debug menu) won't go above 60. I am on a GTX 1060 6GB with the proprietary drivers, Arch Linux and KDE Plasma. I have tried V-Sync on *and* off, but that didn't make a difference. Same goes for fullscreen. My main monitor (the one it fullscreened to) is 1440p@165Hz. How can I make it run at a higher framerate?

**Comment by concubidated** [comment, marked as answer]
Can you verify with glxgears?

If you are seeing 60fps there, can you try 

`__GL_SYNC_TO_VBLANK=0 glxgears`

If that works, then you probabt just need to do the same when your boot sm.

**Reply by puyoxyz** [reply]
Wasn't having the issue with glxgears, but using that environment variable worked for StepMania. Thanks!
